src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -89,63 +89,63 @@ def __init__(
     def initialize_datamodel(
         self, request: DataModelProtoModule.InitDatamodelRequest
     ) -> DataModelProtoModule.InitDatamodelResponse:
-        """initDatamodel rpc of DataModel service."""
+        """InitDatamodel rpc of DataModel service."""
```

Comment:
Is this changed by pre-commit? I added these words in an exclusion list for the docformatter, probably some local sync issue? If I run `pre-commit run --all-files` locally it doesn't report any formatting change.

Reply:
Is this changed by pre-commit? - yes.

`pre-commit run --all-files` does not resolve this issue.